File: sajadv-app/src/app/model/pessoa.ts
export class Pessoa {

  id: number;
  nome: string;
  cpf: string;
  email: string;
  dataNascimento: Date;

  constructor(id: number, nome: string, cpf: string, email: string, dataNascimento: Date) {
    this.id = id;
    this.nome = nome;
    this.cpf = cpf;
    this.email = email;
    this.dataNascimento = dataNascimento;
  }

}

File: sajadv-entity/src/main/java/br/com/portfolio/lsilva/sajadv/entity/Pessoa.java
package br.com.portfolio.lsilva.sajadv.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Column;
import javax.persistence.Entity;
import java.time.LocalDate;

@Data
@Entity
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Pessoa extends BaseEntity {

    @Column(length = 150)
    private String nome;

    @Column(unique = true, length = 14)
    private String cpf;

    @Column(nullable = false, length = 400)
    private String email;

    @Column(name = "data_nascimento")
    private LocalDate dataNascimento;

    private String arquivo;

    private boolean ativo = true;
}
